### DISCUSSION TITLE
[Config Request]

### DISCUSSION THREAD

**Discussion by ImpulsiveLass** [gamehog44/LethalRadiation#8] [Ideas]
Would it be possible to change the time of the radiation damage or no since its based off of the hour of the day as a whole and takes into account how late in the day it was taken?

I'm planning on running this with lethal meltdown but if the facility blows up in 2 hours anyway then 10damage and a little blur per hour is joever.

**Comment by gamehog44** [comment]
Thank you for the suggestion! Unfortunately, making a config for that would change the core concept of the mod. As you said, it goes off the top of the hour and the time since it was pulled. It sounds like it’s best to only use one of the apparatus altering mods.
